hyperspy/drawing/_widgets/line2d.py

```diff
@@ -139,7 +143,6 @@ def _set_axes(self, axes):
         # _set_axes overwrites self._size so we back it up
         size = self._size
         super(Line2DWidget, self)._set_axes(axes)
-        self._pos = np.tile(self._pos, (2, 1))
```

Comment:
Why did you remove this line?

Reply:
It seems that I forgot to push a commit (which is done now), where the `_pos` is backed up and restored just after the call of `_set_axes()` of the parent class. In this case, I would not expect to need to tile `_pos`.
I don't remember the reason for this, I may have to come back later to this. It was not obvious to me what was going on, I may have miss something.